packages/webview_flutter/webview_flutter_wkwebview/lib/src/web_kit/web_kit.dart

```diff
@@ -202,6 +232,22 @@ class WKWebViewConfiguration {
   }
 }
 
+/// The methods for presenting native user interface elements on behalf of a webpage.
+///
+/// Wraps [WKUIDelegate](https://developer.apple.com/documentation/webkit/wkuidelegate?language=objc).
+class WKUIDelegate {
+  /// Indicates a new [WebView] was requested to be created with [configuration].
+  void setOnCreateWebView(
```

Comment:
Why is this a function named `setFoo` rather than a `set foo`? (This may apply to code from previous PRs and I just didn't notice it then.)

Reply:
That's my mistake. It was supposed to have a `Future<void>`. But, I suppose using `set foo` IS more Darty. But it also would be an asynchronous function so I'm not sure which is better. I can't think of an instance where it would need to be awaited though.